src/test/function_test/utils.h

```diff
@@ -30,3 +33,50 @@
             }                                                                                      \
         }                                                                                          \
     } while (0)
+
+inline std::string generate_random_str(uint32_t str_len = 20)
+{
+    static const std::string chars("abcdefghijklmnopqrstuvwxyz"
+                                   "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                                   "1234567890");
+    std::string result;
+    for (int i = 0; i < str_len; i++) {
+        result += chars[dsn::rand::next_u32(chars.size())];
+    }
+    return result;
+}
```

Comment:
consider add the function into `dsn::rand` as `common` function

Reply:
I will do it in the next PR